floyd/cli/experiment.py

```diff
@@ -58,6 +61,10 @@ def init(project_name):
     ExperimentConfigManager.set_config(experiment_config)
     FloydIgnoreManager.init()
 
+    yaml_config = read_yaml_config()
+    if not yaml_config:
```

Comment:
To be safe should this check if the `floyd.yml` file already exists. To avoid overwriting any local changes.

Reply:
`read_yaml_config` checks to see if the config file exists, if not it will return None, which will trigger the config creation code.